src/jit/compiler.h

```diff
@@ -1140,6 +1140,7 @@ struct fgArgTabEntry
     SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
 #endif // defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
 
+#ifdef _TARGET_ARM_
```

Comment:
That's the way we did some of the SIMD-specific methods as well; I think it makes sense.